### DISCUSSION TITLE
Get /blog/[slug].js and /blog/index.js out of next export

### DISCUSSION THREAD

**Discussion by funwhilelost** [vercel/next.js#16534] [Help]
Just upgraded to Next 9 and started using the dynamic routing from the package.  Awesome work!  One thing that's throwing me for a loop is the files coming out of this situation:

`pages/blog/[slug].js` -> `out/blog/post-name.html`
`pages/blog/index.js` -> `out/blog.html`

This is a bit confusing to me because when I use `npx serve out` or `npx http-server out` neither of them will render `http://localhost:8080/blog`.  They're looking for `/out/blog/index.html` which is not being created.  My /blog/index.js source is being exported as `blog.html` which is not one of the fallbacks for either of the basic, static-file servers.

Is there a config option I can set, or a `getStaticPaths` configuration I can give, or something else entirely?

**Comment by Timer** [comment]
You can export use following in `next.config.js`:

```js
module.exports = {
  trailingSlash: true
}
```

**Reply by funwhilelost** [reply]
@Timer Worked great.  Is this documented somewhere I can understand it better?  It's a little opaque to me.

**Reply by Timer** [reply]
https://nextjs.org/docs/api-reference/next.config.js/trailing-slash
